### ISSUE TITLE
Apple Pencil not dismissing modals?

### ISSUE THREAD

**Issue by rjpdev** [sweetalert2/sweetalert2#1786]
My Apple Pencil (v1) won't dismiss modals - either by tapping any buttons or outside the modal - it essentially holds my iPad hostage when browsing with the pencil...

I thought it was an iOS/Safari bug that was somehow being overlooked... then I used my finger.

You can replicate this anywhere that sweetalert2 is being used:

https://sweetalert2.github.io/

Before anyone goes dismissing this because no one browses the internet with an Apple Pencil, I'll have you know, there are dozens of us! Dozens!

**Comment by rjpdev** [comment]
This was fixed after updating to iPadOS 13.1.3!

**Comment by Anubiso** [comment]
This issue is still open - using an iPad Pro with iPadOS 13.7 
The issue lies in the iOSfix for scrolling, as the combination of:

```
container.ontouchstart = (e) => {
    preventTouchMove = shouldPreventTouchMove(e.target)
  }
  container.ontouchmove = (e) => {
    if (preventTouchMove) {
      e.preventDefault()
      e.stopPropagation()
    }
  }
```

prevents the click of the button as you always move a little bit with the pencil instead of a click which would be triggered shortly after ontouchmove. if you try really hard to not move while clicking it recognizes the click and does not call preventDefault.
